src/manifest/mod.ts

```diff
@@ -274,9 +274,11 @@ export class SlackManifest {
   private assignRunOnSlackManifestProperties(manifest: ManifestSchema) {
     const def = this.definition as ISlackManifestRunOnSlack;
 
-    // Run on Slack Apps do not manage access tokens
-    // This is set by default as false
-    manifest.oauth_config.token_management_enabled = false;
+    // Oauth Config
+    // Run On Slack manage their own tokens
+    // if and only if user scopes are included in the manifest
+    manifest.oauth_config.token_management_enabled =
+      manifest.oauth_config.scopes.user != undefined;
```

Comment:
If the user specifies user scopes as an empty array, token management will be enabled. Is this the behaviour we want?

Reply:
@filmaj - Good catch, thank you! token management enabled should remain `false` in that case for run on slack apps. I've updated the SlackManifest + the tests.